terraform/gcp/modules/sigstore/sigstore.tf

```diff
@@ -231,6 +235,8 @@ module "rekor" {
 
   redis_cluster_memory_size_gb = var.redis_cluster_memory_size_gb
 
+  index_database_instance_name = module.mysql.mysql_instance
+
   depends_on = [
```

Comment:
Do you need to add `mysql.mysql_instance` for a dependency?

Reply:
Since this is an [output of the mysql module](https://github.com/sigstore/scaffolding/blob/9abab05d155fa0ae8349c234eb8a609db161cbf0/terraform/gcp/modules/mysql/outputs.tf#L23), the dependency is implicitly created.